tests/digraph/test_strongly_connected.py

```diff
@@ -65,3 +65,55 @@ def test_number_strongly_connected_big(self):
             node = G.add_node(i)
             G.add_child(node, str(i), {})
         self.assertEqual(len(rustworkx.strongly_connected_components(G)), 200000)
+
+
+class TestCondensation(unittest.TestCase):
```

Comment:
The existing test case is excellent! But we need to add one test covering the case where we pass a list to `sccs` as an argument